js/app.js

```diff
@@ -11,17 +11,18 @@ function Project (projectObj) {
   this.synopsis = projectObj.synopsis;
 }
 
-Project.prototype.render = function () {
+Project.prototype.toHtml = function () {
   var $article = $('.template').clone();
   $article.removeClass('template');
   $article.find('h1').text(this.title);
+  $article.find('img').attr('src', this.imageUrl);
   $article.find('a:nth-of-type(1)').prop('href', this.githubUrl);
   $article.find('a:nth-of-type(2)').prop('href', this.deploymentUrl);
   $article.find('p').text(this.synopsis);
   return $article;
 };
 
-function renderBio () {
+function bioToHtml () {
```

Comment:
Nice. You have some extra data in your bioData.js that you do not have on your page. You should totally add it. It would be great on a portfolio to include the languages you know!